Source/ImGui/Private/ImGuiContextProxy.cpp

```diff
@@ -92,6 +92,7 @@ FImGuiContextProxy::FImGuiContextProxy(const FString& InName, int32 InContextInd
 
 	// Start initialization.
 	ImGuiIO& IO = ImGui::GetIO();
+	InputState.IO = IO;
```

Comment:
This is wrong, you're copying the global IO struct. Instead, I recommend storing IO in InputState as a pointer. This causes IO to not work at all.